src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -442,24 +466,28 @@ class SpanDetail extends React.Component<Props, State> {
                   : null}
               </Row>
               <Tags span={span} />
-              {map(span?.data ?? {}, (v, k) => {
-                const key = (k as unknown) as string;
-                const value = (v as unknown) as number;
-                return (
-                  <Row title={key} key={key}>
-                    {SIZE_DATA.has(key) ? (
-                      <React.Fragment>
-                        <FileSize bytes={value} />
-                        {value >= 1024 && (
-                          <span>{` (${JSON.stringify(value, null, 4) || ''} B)`}</span>
-                        )}
-                      </React.Fragment>
-                    ) : (
-                      JSON.stringify(value, null, 4) || ''
+              {allZeroSizes && (
+                <TextTr>
+                  The following sizes were not collected for security reasons. Check if
+                  this host serves <span className="val-string">timing-allow-origin</span>
```

Comment:
```suggestion
                  this host serves <span className="val-string">Timing-Allow-Origin</span>
```

Reply:
Would it be worth linking to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Timing-Allow-Origin ? 🤔

Reply:
That's a good idea, I'll add it in.

Reply:
Updated in 6de3e5d